webapp/src/containers/components/modals/actionModal.tsx

```diff
@@ -5,21 +5,28 @@ import Table from 'react-bootstrap/Table';
 import Form from 'react-bootstrap/Form';
 import './styles.css';
 
+// eslint-disable-next-line import/no-unresolved
+import {Configs} from 'types/plugin/common';
+
 interface Props {
     visible: boolean;
     setVis: React.Dispatch<React.SetStateAction<boolean>>;
```

Comment:
```suggestion
    setVisibility: React.Dispatch<React.SetStateAction<boolean>>;
```
always write full and descriptive variable names.